### DISCUSSION TITLE
JetBrains Extension Allow Toggling of Automatically Show Completions

### DISCUSSION THREAD

**Discussion by christianmagill** [community/community#8313] [Copilot Conversations] [answered]
Sometimes Copilot is just too intrusive. I'd like to be able to toggle the setting "Automatically Show Completions" through a menu item, and ultimately a keyboard shortcut.

**Comment by jansorg** [comment, marked as answer]
This is already possible. There's a setting at `Preferences > Languages & Frameworks > GitHub Copilot`. There's also an action for keyboard use, you can find it at `Tools > GitHub Copilot`. Currently, the setting is an application setting. The next update will make this a project setting and add the possibility to also suppress automatic completions for certain languages only.

**Comment by christianmagill** [comment]
@jansorg, I'd like to be able to toggle that setting with a keyboard shortcut. I don't believe that is currently possible. It would be nice to be able to toggle the automatic completion suggestions off and on without having to dig into the settings. Sometimes it is useful, and sometimes the suggestions get in the way.

**Comment by jansorg** [comment]
@christianmagill You can configure any action with your own keyboard shortcut. There's already a default for the action mentioned above, it's `Ctrl+Alt+Shift+O`. 
Here's where you can set your own shortcut:
![Screenshot_20211202_180940](https://user-images.githubusercontent.com/11473063/144469846-ba9a5272-54bf-4a2a-a70f-686d06296352.png)

**Reply by christianmagill** [reply]
Much thanks! That does exactly what I need.
